packages/react-server-webpack-plugin/src/transforms.ts

```diff
@@ -0,0 +1,25 @@
+import * as t from '@babel/types';
+
```

Comment:
It feels a bit weird to me to do this as an ast transform rather than just using handlebars or something since it's so simple